modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/Sidebar.js

```diff
@@ -71,17 +71,19 @@ const Sidebar = ({children, onClose = noop, open = true}) => {
 		}
 	}, [delay, open]);
 
+	useEffect(() => {
+		const body = document.querySelector('body');
+
+		isOpen
+			? body.classList.add(`sidebar-open`)
+			: body.classList.remove(`sidebar-open`);
+	}, [isOpen]);
+
 	return (
-		<div
-			className={classNames('sidebar-wrapper', {
-				open: isOpen,
-			})}
-		>
-			<div className="sidebar sidebar-light sidebar-sm">
-				<SidebarContext.Provider value={{onClose}}>
-					{children}
-				</SidebarContext.Provider>
-			</div>
+		<div className={`sidebar sidebar-light sidebar-sm`}>
```

Comment:
You don't need a template literal here, or even a JS value.

```js
<div className="sidebar sidebar-light sidebar-sm">
```

Reply:
I was using a variable there in previous versions, forgot to remove it, thx!